entrypoint/validatecluster/validatecluster_test.go

```diff
@@ -38,6 +38,18 @@ var _ = Describe("Test:", func() {
 		testcase.TestServiceNodePort(true, true)
 	})
 
+	It("Checks the healthz endpoints", func() {
```

Comment:
we are only going to add those new checks on validate cluster flow?

Reply:
I have a low level awareness of the surface area of the framework still but I'd be open to checking these endpoints in other areas based on the teams recommendations.

Reply:
got you

so what u can probably do is, go entrypoint by entrypoint checking the `_test.go` files, to see in which flows of tests this would apply nicely, for instance on the upgrades and etcd.

![Screenshot 2024-08-02 at 16 08 16](https://github.com/user-attachments/assets/206161cb-969a-4af2-a61a-21b146885705)

Reply:
? @VestigeJ

Reply:
The only place I think I may add it for now is on the upgrade checks

Reply:
nice ok, lets add there also